### DISCUSSION TITLE
Wake on opening lid not working

### DISCUSSION THREAD

**Discussion by xddxd** [MultimediaLucario/Lenovo-ThinkPad-T440s#25] [Q&A] [answered]
Hi, just upgraded from Big Sur + Sniki's EFI to Monterey with this EFI. Nice work, it's running nicely. However, I have encountered this issue that the laptop won't wake up from sleep when I open the lid, I have to press the power button for it to wake up, keyboard keys don't wake it up either. Is this possible to fix? It's a T440s with an i5-4200U.

**Comment by MultimediaLucario** [comment]
I am currently working on it.

**Reply by MultimediaLucario** [reply]
More importantly how are you able to get macOS to work on a ThinkPad with an i5-4200U? I am still trying to get macOS to work on my brother's X240.

**Reply by xddxd** [reply]
Why wouldn't it work? I've been using macOS on this laptop since Catalina. Is there some specific issue with this CPU?

**Comment by MultimediaLucario** [comment, marked as answer]
Also, Open the Config.plist file go to ACPI, expand the Patch section, open the Item 27 and set the instant wake fix enabled toggle to yes.
